src/utils/src/Thread.c

```diff
@@ -198,6 +213,23 @@ PUBLIC_API STATUS defaultCreateThread(PTID pThreadId, startRoutine start, PVOID
     return retStatus;
 }
 
+PUBLIC_API STATUS defaultCreateThread(PTID pThreadId, startRoutine start, PVOID args)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+
+#if defined(KVS_DEFAULT_STACK_SIZE)
+    CHK_STATUS(defaultCreateThreadWithParams(pThreadId, start, KVS_DEFAULT_STACK_SIZE, args));
+#elif defined(CONSTRAINED_DEVICE)
```

Comment:
Should we document/comment somewhere to point out that a specified default stack size will supersede the constrained device size?

Reply:
Added to producer-c where CONSTRAINED_DEVICE is listed.